libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -508,6 +528,19 @@ export class ActivityHandler extends ActivityHandlerBase {
         }
     }
 
+    /**
+     * Invoked when the bot is sent an Adaptive Card Action Execute.
+     *
+     * @param context the context object for the current turn
+     * @param invokeValue incoming activity value
+     */
+    protected async onAdaptiveCardInvoke(
```

Comment:
`async` isn't necessary 😄

Reply:
It is a convenient way to make a function that throws an error actually return a `Promise.reject`, but I'll clean it up 😄